components/Footer.js

```diff
@@ -9,6 +7,9 @@ import RocketLaunchIcon from "@heroicons/react/20/solid/RocketLaunchIcon";
 import Button from "./Button";
 import Link from "@components/Link";
 import app from "@config/app.json";
+import { PROJECT_NAME } from "@constants/index";
+import { BASE_GITHUB_PROJECT_URL, BASE_GITHUB_URL } from "@constants/index";
```

Comment:
I think this can be in 1 line?

```suggestion
import { BASE_GITHUB_PROJECT_URL, BASE_GITHUB_URL, PROJECT_NAME } from "@constants/index";
```

Reply:
Fixed

Reply:
Thank you 👍